bin/steps/pip-install

```diff
@@ -2,8 +2,16 @@
 
 source $BIN_DIR/utils
 
-export LIBRARY_PATH=/app/.heroku/vendor/lib:$BUILD_DIR/.heroku/vendor/lib:/app/.heroku/python/lib
-export LD_LIBRARY_PATH=$HOME/.apt/usr/lib:/app/.heroku/vendor/lib:$BUILD_DIR/.heroku/vendor/lib:/app/.heroku/python/lib
+export LIBRARY_PATH=${LIBRARY_PATH}:/app/.heroku/python/lib
+export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/app/.heroku/python/lib
+
+# Horrible hack to remove static libraries so that
+# the shared ones are used.
+for LIB in libgmp libyaml ; do
+  if [ -f /app/.apt/usr/lib/x86_64-linux-gnu/${LIB}.so ] ; then
```

Comment:
We've usually (within the Cloud scripts) tried to avoid `;` and use an actual line break instead...but I won't loose any :zzz: over it :wink: